rasa/core/policies/memoization.py

```diff
@@ -55,16 +61,14 @@ def _standard_featurizer(
         # Memoization policy always uses MaxHistoryTrackerFeaturizer
         # without state_featurizer
         return MaxHistoryTrackerFeaturizer(
-            state_featurizer=None,
-            max_history=max_history,
-            use_intent_probabilities=False,
+            state_featurizer=None, max_history=max_history
         )
 
     def __init__(
         self,
         featurizer: Optional[TrackerFeaturizer] = None,
         priority: int = MEMOIZATION_POLICY_PRIORITY,
-        max_history: Optional[int] = None,
+        max_history: Optional[int] = -1,
```

Comment:
`-1 ` is a magic value which has a special meaning. We should use a constant for this.